mslib/msui/mscolab_chat.py

```diff
@@ -532,7 +534,7 @@ def setup_image_message_box(self):
                               self.attachment_path.replace('\\', '/').split('colabdata')[1])
         else:
             img_url = urljoin(self.chat_window.mscolab_server_url, self.attachment_path)
-        data = requests.get(img_url, timeout=tuple(config_loader(dataset="MSCOLAB_timeout"))).content
+        data = requests.get(img_url, timeout=MSCOLAB_TIMEOUT).content
```

Comment:
Unrelated issue: this accesses the responses content without first checking for success.

Reply:
yes, first step I wanted to solve only timeout configurations. There are likly more like that.